cypress/e2e/models/migration/migration-waves/migration-wave.ts

```diff
@@ -260,23 +260,75 @@ export class MigrationWave {
         cy.get("button").contains(stakeHolderGroupName).click();
     }
 
+    private formatDate(date: Date): string {
+        const day = String(date.getDate()).padStart(2, "0");
+        const month = String(date.getMonth() + 1).padStart(2, "0"); //January is 0!
+        const year = date.getFullYear();
+
+        return month + "/" + day + "/" + year;
+    }
+
     public expandActionsMenu() {
-        cy.contains(this.name)
-            .parents("tr")
-            .within(() => {
-                cy.get(MigrationWaveView.actionsButton).then(($btn) => {
-                    if ($btn.attr("aria-expanded") === "false") {
-                        $btn.trigger("click");
+        let targetName: string | undefined;
+        let targetStartDate: string | undefined;
+        let targetEndDate: string | undefined;
+
+        if (this.name) {
+            targetName = this.name;
+            cy.contains(targetName)
+                .parents("tr")
+                .within(() => {
+                    cy.get(MigrationWaveView.actionsButton).then(($btn) => {
+                        if ($btn.attr("aria-expanded") === "false") {
+                            $btn.trigger("click");
+                        }
+                    });
+                });
+        } else if (this.startDate && this.endDate) {
+            targetStartDate = this.startDate ? this.formatDate(this.startDate) : undefined;
+            targetEndDate = this.endDate ? this.formatDate(this.endDate) : undefined;
```

Comment:
The `if`  is already verifying that `startDate` and `endDate` aren't null or undefined, so there is no need to also check it in the ternary operator.

        targetStartDate = this.formatDate(this.startDate);
        targetEndDate = this.formatDate(this.endDate);

Reply:
These two variables can be declared here as `const` so there is no need o declare them as `let` above